.travis.yml

```diff
@@ -439,7 +439,7 @@ cache:
 
 before_install:
   # NUM_PROCESSORS was set to 1 due to problems in parallel launch of unit tests on Mac platform
```

Comment:
This comment doesn't seem to apply any more.

Since it isn't related to the issue on Windows, maybe this restriction isn't needed at all any more?

Reply:
I pushed two commits. The first just fixes the comment. The second removes the MacOS restriction. If it works, I'll keep both, else I'll drop the second one.

Reply:
Looks like it worked: https://travis-ci.com/github/ximinez/rippled/builds/184981619